src/igroupadmin.cpp

```diff
@@ -466,14 +466,25 @@ void usageMain() {
         "certain administrative functions.  It can also be used by regular",
         "users to list groups ('lg') and by the admin for all operations.",
         " ",
+        "After making a new group they will themselves manage, a groupadmin's
+        "first action should be to add their own username to the group.",
+        "They will from that point forward be able to add users to, and remove",
+        "them from, the group.  Those operations permit, as well, the transfer",
+        "of responsibility for the group to another groupadmin.",
+        " ",
+        "Currently, a groupadmin may not remove groups or users from the zone."
```

Comment:
I'm definitely guilty of adding words like this to my stuff as well, but I think "Currently, " isn't adding much here. Consider removing it.

Missing a comma at the end of this line.